frontend/src/pages/welcome/welcome.tsx

```diff
@@ -0,0 +1,33 @@
+import '../../assets/css/typography.scss';
+
+import { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { BusinessCard, CustomerCard } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/enums.js';
+
+import styles from './styles.module.scss';
+
+const Welcome: React.FC = () => {
+  const navigate = useNavigate();
+
+  const handleClick = useCallback(() => {
+    navigate(`${AppRoute.SIGN_UP}`);
+  }, [navigate]);
+
+  return (
+    <section className={styles.section}>
+      <h2 className={`h4 ${styles.title}`}>
+        Welcome to the <span className={styles.accent}>TowHub</span>! Let&apos;s
+        get to know each other better
+      </h2>
+
+      <ul className={styles.list}>
+        <BusinessCard onClick={handleClick} />
```

Comment:
Please make sure they are responsible and do the screenshot from mobile view next time

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/110409579/a6206152-3313-43f2-9cf7-ab9906084d52)
tablet

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/110409579/10379b7a-e17f-4a9e-a07b-00cb075af812)
320px

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/110409579/70571f06-bab7-4e40-95da-27d86159b48f)
768px

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/110409579/5602b891-363e-4269-bfc7-7a27a31a670d)
desktop